src/js/components/Layout/Logo.jsx

```diff
@@ -4,19 +4,29 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Link, withRouter } from 'react-router-dom';
 
-import OpenboxesLogo from 'components/Layout/img/openboxes_logo.jpg';
-
 
 const Logo = ({
   logoUrl,
+  defaultLogoUrl,
 }) => {
-  const [imgSrc, setImgSrc] = useState(null);
+  const [imgSrc, setImgSrc] = useState('');
+  const [defaultLogo, setDefaultLogo] = useState('');
   useEffect(() => {
-    setImgSrc(logoUrl || OpenboxesLogo);
+    if (logoUrl) {
+      setImgSrc(logoUrl);
+    }
   }, [logoUrl]);
 
+  useEffect(() => {
+    if (defaultLogoUrl) {
+      setDefaultLogo(defaultLogoUrl);
+    }
+  }, [defaultLogoUrl]);
+
   const handleError = () => {
-    setImgSrc(OpenboxesLogo);
+    if (defaultLogo) {
+      setImgSrc(defaultLogo);
+    }
```

Comment:
this is why we needed this additional `defaultLogoUrl` from `AppContext` - if the `img` handles an error (url is correct, but it does not find an image on this path, it will set default logo then. If we didn't have `defaultLogoUrl`, it would print `alt` from `img` in this case.